src/libtest/stats.rs

```diff
@@ -11,7 +11,7 @@
 #![allow(missing_docs)]
 #![allow(deprecated)] // Float
 
-use std::cmp::Ordering::{self, Less, Greater, Equal};
+use std::cmp::Ordering::{self, Equal, Greater, Less};
```

Comment:
Well, the rule can be generalized to all `enum`s by sorting variants by declaration order. This has the added benefit of inheriting any grouping that the author put in. Taking a few random examples from MIR to show how this rule would provide benefit:

- `rustc::mir::repr::TerminatorKind` has a few intuitive groups. `Goto`, `If`, `Switch`, and `SwitchInt` are basic branches, going to one of a list of possible targets based on a simple condition. `Resume` and `Return` both signal exiting from some sort of scope, and `Drop`, `DropAndReplace`, `Call`, and `Assert` are all not inherently terminators, more like statements that can panic. By sorting by definition order, those groups would all be imported next to each other.
- `rustc::mir::repr::Lvalue` is similarly grouped: `Var`, `Temp`, and `Arg` are function-local, `Static` and `ReturnPointer` represent externally created variables, and `Projection` is its own thing.
- `rustc::mir::repr::BorrowKind` does not really have a division into groups. However, it is ordered by increasing permissions: `Shared` pointers can't be guaranteed unique access and can't mutate, `Unique` pointers form the in-between of unique access without mutability, and `Mut` pointers have all the permissions. Importing them in this order makes sense because it is the natural order to view the options in.

Essentially, developers already sort their `enum`s in a logical, helpful way, and it makes sense to reuse that for import order.